.github/workflows/ci-schedule.yml

```
@@ -3,6 +3,9 @@ on:
   schedule:
     # Run this workflow "At 18:00 UTC on Sunday and Saturday"
     - cron: '0 18 * * 0,6'
+
+permissions:
+  contents: read # Necessary for fetching the repository contents, enabling actions like actions/checkout@v4 to work properly.    
```

```suggestion
  contents: read
```

there is no need to add comment to `read` permissions, if have to, it should also be consistent across workflows